### PR TITLE
improve syntax and resolve app breaking bugs

### PR DESCRIPTION
**Pull request by bencoullie**
Related to: https://github.com/EricTurf/react-sudoku/issues/26 👍 

Hope this you don't mind the syntax changes. I just combed the whole file for potential issues and unnecessary syntax and cleaned it up a bit.

This allows the app to load and thus gives contributors a better starting point.

### CODE REVIEW

File: src/dialog.jsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import React from 'react';
```
Comment (bencoullie) [inline src/dialog.jsx]: Happy to go back and add in  semicolons if you're particular about that.

File: src/dialog.jsx [hunk 2]
```diff
@@ -73,22 +66,12 @@ render() {
...
   }
 }
 
-}
-
-
-Dialog = props => {
-  const {
-    header, message, isOpen, stateManager,completionTimeMessage
-  } = props;
```
Comment (bencoullie) [inline src/dialog.jsx]: This was the mail culprit here ☝️

Comment (EricTurf) [reply src/dialog.jsx]: I see, thank you for the fix, I believe I screwed things up with the browser conflict resolver. Seems like you have a conflict so if that is fixed I can merge it :)

**Comment by bencoullie** [comment]
Looks like this is already fixed 👍

**Pull request closed without merging**